backend/pkg/config/config.go

```diff
@@ -77,6 +79,7 @@ func LoadConfig() (*Config, error) {
 	config.DBHost = viper.GetString(DBHostEnvVar)
 	config.DBPort = viper.GetString(DBPortEnvVar)
 	config.DBName = viper.GetString(DBNameEnvVar)
+	config.DBSSLMode = viper.GetString(DBSSLModeEnvVar)
```

Comment:
please set default value to `disable` to be backward compatible
defaults are being set in backend/cmd/backend/main.go

Reply:
definitely!